src/components/DropDown/DropDown.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { useEffect } from 'react';
+import styled from 'styled-components';
+import '@/styles/font.css';
+import { useState } from 'react';
+import arrowTopIcon from '@/assets/icons/arrow_top.svg';
+import arrowBottomIcon from '@/assets/icons/arrow_bottom.svg';
+
+// TODO: option리스트를 배열로 받기
+const options = ['옵션예시1번', '옵션예시2번', '옵션예시3번', '옵션예시4번'];
+
+function DropDown() {
+  const [selected, setSelected] = useState(options[1]);
+  const [isActive, setIsActive] = useState(false);
+  return (
+    <Style>
+      <DropDownBtn $isActive={isActive} onClick={() => setIsActive((prev) => !prev)}>
+        {selected}
+        {isActive ? (
+          <ArrowIcon src={arrowBottomIcon} alt="아래화살표" />
+        ) : (
+          <ArrowIcon src={arrowTopIcon} alt="위화살표" />
+        )}
+      </DropDownBtn>
+      {isActive && (
+        <DropDownContentsWrapper>
+          {options.map((option) => (
+            <DropDownContent
+              key={option}
+              onClick={() => {
+                setSelected(option);
+                setIsActive(false);
+              }}
+            >
+              {option}
+            </DropDownContent>
+          ))}
+        </DropDownContentsWrapper>
+      )}
+    </Style>
+  );
+}
+const Style = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+`;
```

Comment:
저희가 의논한대로 감싸는 컴포넌트는 컴포넌트 이름 + container라고 지으면 됩니다.
즉 여기는
```suggestion
const DropdownContainer = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
`;
```

Reply:
로 해주시면 좋을듯 합니다.